patches/tModLoader/Terraria/Localization/Content/pl-PL/tModLoader.json

```diff
@@ -20,17 +19,17 @@
 		"ModsModPacks": "Paczki modów",
 		"ModsDisable": "Wyłącz",
 		"ModsEnable": "Włącz",
-		// "ModsOpenConfig": "Click to open config",
+		"ModsOpenConfig": "Kliknij, aby otworzyć konfiguracje",
 		"ModOldWarning": "Stary mod, aktywuj na własną odpowiedzialność",
 		"ModDependencyClickTooltip": "Ten mod wymaga moda: {0}\n (kliknij by włączyć)",
 		"ModDependencyModsNotFound": "Nie znaleziono następujących modów: {0}",
-		// "ModDidNotFullyUnloadWarning": "This mod did not fully unload during last unload.\n(This is a message for the mod creator)",
+		"ModDidNotFullyUnloadWarning": "Ten mod nie został w pełni rozładowany podczas ostatniego rozładunku.\n(This is a message for the mod creator)",
```

Comment:
Ten mod nie został w pełni rozładowany podczas ostatniego rozładunku.\n(To jest wiadomość dla twórcy moda)